lib/src/realm_property.dart

```diff
@@ -46,7 +46,6 @@ enum RealmPropertyType {
   uuid,
 }
 
-/// A annotation class used to define Realm data model classes and their properties
 class RealmProperty {
```

Comment:
Do we need this anymore?

Reply:
SchemaProperty is RealmProperty. I can merge the two classes or I could change this to a private base class.